### DISCUSSION TITLE
When both startAxis and EndAxis is used with axisValuesOverrider

### DISCUSSION THREAD

**Discussion by Curwe** [patrykandpatrick/vico#440] [Questions]
Hi.
I have an app where I show both a ColumnChart and LineChart. 
The ColumChart looks like this 
```
@Composable
private fun columnChart(chartData: ChartData) = columnChart(
    columns = listOf(columnComponent, unSelectColumnComponent),
    spacing = COLUMN_SPACING.dp,
    targetVerticalAxisPosition = AxisPosition.Vertical.Start,
    axisValuesOverrider = AxisValuesOverrider.fixed(
        maxY = chartData.axisMaxValue?.toFloat(),
        minY = 0
    )
)
```

And LineChart like this 
```
@Composable
private fun lineChart(chartData: ChartData) = lineChart(
    lines = listOf(
        LineChart.LineSpec(
            lineColor = Color.Black.toArgb(),
            lineThicknessDp = 1.5f,
        )
    ),
    targetVerticalAxisPosition = AxisPosition.Vertical.End,
    axisValuesOverrider = AxisValuesOverrider.fixed(
        maxY = chartData.axisMaxValue?.toFloat(),
        minY = chartData.axisMinValue // This can be negative 
    )
)
```

The problem now is the startAxis and endAxis. Both look like this (the * is Start or End in code)-> 
```
@Composable
private fun *Axis() = remember*Axis( 
    itemPlacer = remember { AxisItemPlacer.Vertical.default(4) },
    label = axisLabelComponent(color = Color.ChartLabelColor, textSize = labelSize),
    valueFormatter = { y, _ -> y.toInt().toString() },
    axis = null,
    tick = null,
    guideline = axisGuidelineComponent(
        shape = Shapes.pillShape,
        color = Color.ChartGuidelineColor
    ),
)
```

So the problem now is that the Bottom line value on the startAxis sometimes does not start at 0. And The Bottom line value for endAxis never show a negativ, it's always 0. 

I want them to be independent from each other, so startAxis starts at minY from ColumChart and ends at maxY. And same for LineChart. 

Any way to get this working? 
Attaching image to show how it looks today. In the image the line data is negative, but as you see it still starts from 0 when I want it to start from the negativ value.

![MicrosoftTeams-image (5)](https://github.com/patrykandpatrick/vico/assets/135611838/c98fe51c-ac5d-4b53-bc54-cc531f01335e)

**Comment by patrickmichalik** [comment]
Hello! Could you please provide a complete, runnable example? To accurately determine what’s going on, we need to be able to reproduce the problem. A full example would be particularly helpful here because `ChartData` isn’t a Vico API. Could you also let us know which version of Vico you’re using?

**Reply by Curwe** [reply]
I'm running vico v1.12.0

I'll need some time to setup a example project tho.

**Reply by patrickmichalik** [reply]
Thanks for sharing the Vico version. Regarding the example, that’s perfectly fine. Please take your time.
